docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,280 @@
+---
+title: NGINX Ingress Controller and Istio Service Mesh
+description: |
+  Use NGINX Ingress Controller with Istio Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-889"
+---
+
+## Using NGINX Ingress Controller with Istio service mesh    
+
+NGINX Ingress Controller can now be used as the ingress controller for applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX IC provides on Istio-based environments without resorting to any workarounds.

```

Comment:
```suggestion
NGINX Ingress Controller can now be used as the ingress controller for applications running inside an Istio service mesh. This allows you to continue using the advanced capabilities that NGINX IC provides on Istio-based environments without resorting to any workarounds.
```suggestion
NGINX Ingress Controller can now be used as the ingress controller for applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX IC provides on Istio-based environments without resorting to any workarounds.

```
```